From c80812352d63bd817133f2171d697da9ac805f98 Mon Sep 17 00:00:00 2001
From: "Liu,Dalei" <daleix.liu@intel.com>
Date: Fri, 18 Dec 2015 11:21:57 +0800
Subject: [PATCH 2/3] ov8858: enable embedded meta data

Enable ov8858 embedded data on BTNS.

Change-Id: I37571fa1362238bf468926a9f559b8f1f2d0ff0e
Original-Tracked-On: #H1504125960
Tracked-On:https://jira01.devtools.intel.com/browse/OAM-13813
Signed-off-by: Liu,Dalei <daleix.liu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/90823
Approver: Bin B Han <bin.b.han@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Reviewed-by: HaiqiangX Yao <haiqiangx.yao@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Build: Dessert M CHT FFD (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Signed-off-by: Yang Fan F <fan.f.yang@intel.com>
Reviewed-on: https://android.intel.com:443/452478
---
 drivers/external_drivers/camera/drivers/media/i2c/ov8858_btns.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov8858_btns.h b/drivers/external_drivers/camera/drivers/media/i2c/ov8858_btns.h
index 9233d8b..74937ba 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov8858_btns.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov8858_btns.h
@@ -631,6 +631,7 @@ static struct ov8858_reg ov8858_BasicSettings[] = {
 	{OV8858_8BIT, 0x4300, 0xFF}, /* clip_max[11:4] = 0xFFF */
 	{OV8858_8BIT, 0x4301, 0x00}, /* clip_min[11:4] = 0 */
 	{OV8858_8BIT, 0x4302, 0x0F}, /* clip_min/max[3:0] */
+	{OV8858_8BIT, 0x4307, 0x31}, /* Unknown */
 	{OV8858_8BIT, 0x4316, 0x00}, /* CTRL16 = default */
 	{OV8858_8BIT, 0x4503, 0x18}, /* Unknown */
 /*	{OV8858_8BIT, 0x4500, 0x38}, */ /* Unknown */
-- 
1.9.1

